packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -290,14 +286,32 @@ export const LivenessCameraModule = (
     );
   }
 
+  const shouldShowCenteredLoader = isInitCamera || isInitWebsocket;
+
   // We don't show full screen camera on the pre check screen (isStartView/isWaitingForCamera)
   const shouldShowFullScreenCamera =
-    isMobileScreen && !isStartView && !isWaitingForCamera;
+    isMobileScreen &&
+    !isStartView &&
+    !isWaitingForCamera &&
```

Comment:
nit but could we remove `isWaitingForCamera` here since it's a subset of `isInitCamera`?
```suggestion
```

Reply:
good catch, tested locally and this works!